enterprise_access/apps/content_assignments/api.py

```diff
@@ -50,15 +53,33 @@ def get_assignments_for_configuration(
     return queryset
 
 
-def get_assignments_by_learner_email_and_content(
+def get_assignments_for_admin(
```

Comment:
Not changing the behavior of this method or any of its callers, just updating its name to better differentiate it from `get_assignments_for_learner()`